safety/cli.py

```diff
@@ -74,19 +76,28 @@ def check(key, db, json, full_report, bare, stdin, files, cache, ignore, output,
             sys.exit(-1)
     try:
         vulns = safety.check(packages=packages, key=key, db_mirror=db, cached=cache, ignore_ids=ignore, proxy=proxy_dictionary)
-        output_report = report(vulns=vulns, 
-                               full=full_report, 
-                               json_report=json, 
+        output_report = report(vulns=vulns,
+                               full=full_report,
+                               json_report=json,
+                               xml_report=xml,
                                bare_report=bare,
-                               checked_packages=len(packages), 
-                               db=db, 
+                               checked_packages=len(packages),
+                               db=db,
                                key=key)
 
         if output:
-            with open(output, 'w+') as output_file:
-                output_file.write(output_report)
+            if xml:
```

Comment:
I believe this logic must be part of `safety.formatter.report`. My understanding is that you did put this here because then we could have more than one output, which sounds great, but then we would need some extra refactoring.